tsconfig.json

```
@@ -34,7 +34,7 @@
       "AppInstalation",
       "AppBundle",
       "AppSignedRequest",
-      "AppSigningSecret",
+      "AppDetails",
```

maybe also reorder things alphabetical

```suggestion
      "AppSigningSecret",
      "AppDetails",
```